app/selectors/transactionController.ts

```diff
@@ -6,11 +6,16 @@ import { selectPendingSmartTransactionsBySender } from './smartTransactionsContr
 const selectTransactionControllerState = (state: RootState) =>
   state.engine.backgroundState.TransactionController;
 
-const selectTransactionsStrict = createSelector(
+export const selectTransactionsStrict = createSelector(
```

Comment:
```suggestion
const selectTransactionsStrict = createSelector(
```
I guess forgotten while doing tests?